src/Control/Retry.hs

```diff
@@ -31,7 +32,19 @@ module Control.Retry
       RetryPolicyM (..)
     , RetryPolicy
     , retryPolicy
-
+    , RetryStatus
+    -- ** Fields for 'RetryStatus'
+    , rsIterNumber
+    , rsCumulativeDelay
+    , rsPreviousDelay
+    , defaultRetryStatus
+
+    -- ** Lenses for 'RetryStatus'
+    , rsIterNumberL
+    , rsCumulativeDelayL
+    , rsPreviousDelayL
```

Comment:
Good catch! Thats embarassing. Forgot to commit those lenses. Just updated the PR.